modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -242,21 +242,18 @@
         </classes>
     </test>
 
-    <test name="is-tests-default-configuration-ldap" preserve-order="true" parallel="false" group-by-instances="true">
+    <test name="is-test-adaptive-authentication-graaljs" preserve-order="true" parallel="false" group-by-instances="true">
         <classes>
-            <class name="org.wso2.identity.integration.test.base.LDAPUserStoreInitializerTestCase"/>
-            <class name="org.wso2.identity.integration.test.user.mgt.ReadWriteLdapBasedUserMgtTestCase"/>
-            <class name="org.wso2.identity.integration.test.user.mgt.ReadWriteLDAPUserStoreManagerTestCase"/>
+            <class name="org.wso2.identity.integration.test.auth.GraalJsAdaptiveScriptInitializerTestCase"/>
+            <class name="org.wso2.identity.integration.test.auth.AdaptiveScriptTemporaryClaimPersistenceTestCase"/>
+            <class name="org.wso2.identity.integration.test.auth.SecondaryStoreUserLoginTestCase"/>
         </classes>
     </test>
 
-    <test name="is-tests-uuid-user-store" preserve-order="true" parallel="false" group-by-instances="true">
+    <test name="is-test-adaptive-authentication-graaljs-with-restart" preserve-order="true" parallel="false" group-by-instances="true">
+        <parameter name="scriptEngine" value="graaljs"/>
         <classes>
-            <!-- This initializer test should run before running other tests. This will copy the relevant services that
-                 needs to run the new test cases.-->
-            <class name="org.wso2.identity.integration.test.user.mgt.uuid.UUIDUserManagerInitializerTestCase"/>
-            <class name="org.wso2.identity.integration.test.user.mgt.uuid.ReadWriteLDAPUUIDUMTestCase"/>
-            <class name="org.wso2.identity.integration.test.user.mgt.uuid.JDBCUUIDUMTestCase"/>
+            <class name="org.wso2.identity.integration.test.auth.RiskBasedLoginTestCase" />
```

Comment:
No need to add this to a new test suite

Reply:
Facing the same error as before when adding this to the top of `is-test-adaptive-authentication-nashorn` as well. Hence keeping it as a separate test.